guides/source/caching_with_rails.md

```diff
@@ -465,6 +465,10 @@ is often faster than waiting more than a second to retrieve it. Both read and
 write timeouts default to 1 second, but may be set lower if your network is
 consistently low latency.
 
+By default, the cache store will not attempt to reconnect to Redis if the
+connection fails during a request. If you experience frequent disconnects you
+may wish to enable reconnect attempts.
+
```

Comment:
Suggests that … maybe it should reconnect by default 😊

Reply:
FWIW, reconnect_attempts: 1 definitely improved performance in my Heroku app. You know, if anecdotal data points are your thing 🙂